src/pretty.rs

```diff
@@ -12,15 +12,15 @@ fn min_interpolate_sign(text: &str) -> usize {
     reg.find_iter(text)
         .map(|m| {
             let d = m.end() - m.start();
-            // We iterate over all sequences `%+{` and `"%+m`, which could clash with the interpolation
+            // We iterate over all sequences `%+{` and `"%+`, which could clash with the interpolation
             // syntax, and return the maximum number of `%` insead each sequence.
             //
-            // For the case of a closing delimiter `"%m`, we could actually be slightly smarter as we
+            // For the case of a closing delimiter `"%`, we could actually be slightly smarter as we
             // don't necessarily need more `%`, but just a different number of `%`. For example, if the
-            // string contains only one `"%%m`, then single `%` delimiters like `m%"` and `"%m` would
+            // string contains only one `"%%`, then single `%` delimiters like `m%"` and `"%` would
             // be fine. But picking the maximum
             //
-            // TODO: Improve the `"%*m` case if necessary.
+            // TODO: Improve the `"%*` case if necessary.
```

Comment:
I think that comment got messed up way back in 65f1292349e291650ee589315f9cb467ae3caa19. Seems as good a place as any to fix that.